templates/nscd.conf.erb

```diff
@@ -72,8 +72,8 @@
 	shared			hosts		yes
 	max-db-size		hosts		33554432
 
-<% unless @osfamily == 'RedHat' and @operatingsystemrelease.to_f < 6.0 -%>
-	enable-cache		services	<%= @nscd_service_cache %>
+<% unless @osfamily == 'RedHat' and @operatingsystemmajrelease == '5' -%>
```

Comment:
There are a lot of commented out lines in the template.  I assume this is because it was originally just a straight copy of whatever the OS installs. In which case, would it make any sense for there to be a separate template per major EL version?  It might be quite nice if, by default, you got no changes (other than a 'managed by puppet header comment'?) from the OS default config file until you start overriding parameters.